src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -81,12 +70,7 @@
     @Advanced
     @Argument(fullName = "indelSizeToEliminateInRefModel", shortName = "ERCIS", doc = "The size of an indel to check for in the reference model", optional = true)
     public int indelSizeToEliminateInRefModel = 10;
-    
-    /**
-     * Bases with a quality below this threshold will reduced too the minimum usable qualiy score (6).
-     */
-    @Argument(fullName = "base_quality_score_threshold", shortName = "bqst", doc = "Base qualities below this threshold will be reduced to the minimum (" + QualityUtils.MIN_USABLE_Q_SCORE + ")", optional = true)
-    public byte BASE_QUALITY_SCORE_THRESHOLD = PairHMM.BASE_QUALITY_SCORE_THRESHOLD;
```

Comment:
Spoiler alert: the PCR error model doesn't really do anything right now anyway.

Kris originally used the same error mode as HC, but there were still a lot of 3bp deletions (80% sure it was deletions not insertions) in the M2 calls so he cranked it up, but it doesn't so much and we just filter those out after th fact anyway.